config/routes.rb

```diff
@@ -6,5 +6,6 @@
   get '/breeds', to: "breeds#index"
   get '/breeds/:name', to: "breeds#show", as: "breed"
   get '/cats/:id', to: "cats#show"
+  get '/cats', to: "cats#index"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.